src/test/java/org/springframework/data/neo4j/integration/imperative/RepositoryIT.java

```diff
@@ -1892,6 +1868,18 @@ void deleteAllEntities(@Autowired PersonRepository repository) {
 			assertThat(repository.existsById(id2)).isFalse();
 		}
 
+		@Test // DATAGRAPH-1428
+		void deleteAllById(@Autowired PersonRepository repository) {
+
+			PersonWithAllConstructor person3 = new PersonWithAllConstructor(id1, TEST_PERSON1_NAME,
```

Comment:
What's the purpose of that instance? It has the same id as person 1 and you just retrieve it back again in the delete below.